renderer/pages/home.js

```diff
@@ -1,5 +1,5 @@
 /* global require */
-import React from 'react'
+import React, { useState, useCallback, useEffect } from 'react'
```

Comment:
```suggestion
import React, { useState, useCallback } from 'react'
```

seems unused in this file?

Reply:
(Is there a tool we can run to detect and fix these issues automatically?)

Reply:
removed. We already have `eslint` which should detect this. My atom setup seems to be messed up and hasn't been showing this in some cases.